src/doc/tarpl/casts.md

```diff
@@ -0,0 +1,58 @@
+% Casts
+
+Casts are a superset of coercions: every coercion can be explicitly invoked via
```

Comment:
I would start the section by saying what a cast looks like (and thus it is explicit, cf coercions).